scylla/src/transport/connection.rs

```diff
@@ -1515,6 +1517,11 @@ async fn maybe_translated_addr(
     }
 }
 
+/// Opens a connection and performs its setup on CQL level:
+/// - performs OPTIONS/STARTUP handshake (chooses desired connections options);
+/// - registers for all event types using REGISTER request (if this is control connection).
+///
+/// At the beginning, translates node's address, if it is subject to address translation.
 pub(crate) async fn open_connection(
     endpoint: UntranslatedEndpoint,
     source_port: Option<u16>,
```

Comment:
I know that `open_named_connection` is later removed and the name is taken from identity, so this is not important, but: open_connection always sets name to `scylla-rust-driver` and it is the only caller of `open_named_connection`, so driver name will always be `scylla-rust-driver` - however previous commit claimed to change default name. How is it changed if it is never used? Idk, maybe I'm reading the code incorrectly.

Reply:
That's a correct observation. I'll adjust the code correspondingly.